sxw.c

```diff
@@ -33,9 +30,10 @@
  *         defined with double vs single precision and take
  *         appropriate casting measures.
  *	07-16-12 (DLM) - made a ton of changes to try and
- *          get it to compile with the new updated version of soilwat (version 23) */
-/********************************************************/
-/********************************************************/
+ *          get it to compile with the new updated version of soilwat (version 23)
```

Comment:
we are no longer running version 23. change to SOILWAT2

Reply:
Addressed by commit 984bd13.